test/controllers/content_items_controller_test.rb

```diff
@@ -219,14 +219,14 @@ class ContentItemsControllerTest < ActionController::TestCase
   end
 
   test "renders translated content items in their locale" do
-    content_item = content_store_has_schema_example("case_study", "translated")
+    content_item = content_store_has_schema_example("news_article", "news_article_news_story_translated_arabic")
```

Comment:
This will also need to be updated to a different document type

Reply:
Yeah, I think I'd rather leave this one until I do news_articles. There aren't any more examples with translations, so it would be useful to put if off for another two weeks!